app/src/test/java/io/gnosis/safe/ui/settings/owner/intro/OwnerGenerateViewModelTest.kt

```diff
@@ -0,0 +1,49 @@
+package io.gnosis.safe.ui.settings.owner.intro
+
+import io.gnosis.safe.MainCoroutineScopeRule
+import io.gnosis.safe.R
+import io.gnosis.safe.TestLifecycleRule
+import io.gnosis.safe.utils.MnemonicKeyAndAddressDerivator
+import io.mockk.*
+import org.junit.Rule
+import org.junit.Test
+import pm.gnosis.mnemonic.Bip39
+import pm.gnosis.mnemonic.Bip39.Companion.MIN_ENTROPY_BITS
+import pm.gnosis.model.Solidity
+import java.math.BigInteger
+
+class OwnerGenerateViewModelTest {
```

Comment:
This is unrelated to the rest of the PR? I don't mind, if it is.

Reply:
it is here to increase test coverage